README.md

```diff
@@ -31,21 +43,44 @@ methods made available via:
 import org.scalacheck.ops._
 ```
 
-## scalacheck-ops_1-13
+# Installation
+
+### Compatibility
 
-The additional `scalacheck-ops_1-13` library is compiled against 
-ScalaCheck 1.13.x branch because it contains some binary 
-incompatibilities. Specifically, when mixing this library with ScalaTest
-3.x you might notice the following exception:
+**NOTE** Version 2.x and above **requires** JDK >8 and Scala >2.11 as this 
+library expects the java.time standard library module.
+
+**NOTE** As of `scalacheck-ops` 2.x and above, the version of scalacheck is always
+included in the artifact name. Prior to this change, scalacheck-ops with
+no version suffix would pull in ScalaCheck version 1.12.6. 
+
+| Version Limit | ScalaCheck | Artifact Name       | Supported JDK | Supported Scala |
+| :-----------: | :--------: | :-----------------: | :-----------: | :-------------: |
+| 1.x < x < 2.0 |     1.12.6 | scalacheck-ops      | 6 & 8         | 2.10 & 2.11     |
+| x > 2.0       |     1.12.6 | scalacheck-ops_1-12 | 8             | 2.11            |
+| 1.5 < x < 2.0 |     1.13.4 | scalacheck-ops_1-13 | 6 & 8         | 2.10 & 2.11     |
+| x > 2.0       |     1.13.4 | scalacheck-ops_1-13 | 8             | 2.11            |
+| x > 2.0       |     1.14.0 | scalacheck-ops_1-14 | 8             | 2.11            |
```

Comment:
When I usually see tables like this on people's pages I think they tend to put the newest artifact at the top.

Also if the artifact is the "output" then I'd make it either last or first.